packages/api/src/routers/GraphQL/resolvers/Health.ts

```diff
@@ -0,0 +1,17 @@
+import { Int, ObjectType, Query, Resolver, Field } from "type-graphql";
+
+@ObjectType()
+class Health {
+  @Field(() => Int)
+  ok: number;
+}
+
+@Resolver(of => Health)
+class HealthResolver {
+  @Query(returns => Health)
+  async health(): Promise<Health> {
+    return { ok: 1 };
+  }
+}
```

Comment:
We need it for resolvers with no queries but it's also useful to be able to run a simple query to see if the server is up and running.